frontend/utils.mjs

```diff
@@ -48,18 +48,37 @@ export function copyPublicFolder() {
         }
     })
 }
-
-export function copyIndexHtml(from = 'src/index.html', to = 'dist/index.html', entry = 'index') {
+export function copyIndexHtml(from = 'src/index.html', to = 'dist/index.html', entry = 'index', chunks = {}) {
     const timestamp = new Date().valueOf()
+
     fse.writeFileSync(
         path.resolve(__dirname, to),
-        fse
-            .readFileSync(path.resolve(__dirname, from), { encoding: 'utf-8' })
-            .replace(
-                '</head>',
-                `<script type="module" src="${isDev ? jsURL : ''}/static/${entry}.js?${timestamp}"></script>\n` +
```

Comment:
We're serving these assets (and chunks) with `Cache-Control: public, max-age=0`
<img width="768" alt="Screenshot 2021-11-09 at 09 55 19" src="https://user-images.githubusercontent.com/984817/140902979-0b965132-993e-4ac2-a7b6-babebd4c3571.png">
<img width="766" alt="Screenshot 2021-11-09 at 09 55 10" src="https://user-images.githubusercontent.com/984817/140902983-bc44d476-82b4-4a0f-a2e5-780578252d33.png">
<img width="611" alt="Screenshot 2021-11-09 at 09 55 27" src="https://user-images.githubusercontent.com/984817/140902969-9425768b-2889-4d63-9587-4e0e9ac3ab94.png">

So, browsers aren't actually caching this. And even though we're serving an e-tag in responses our requests don't have `if-match` or `if-none-match` so we're still downloading the content even though we have it in memory.

I don't think that necessarily blocks this PR but we should be serving these assets with `Cache-Control: max-age=365000000, immutable` so that browsers know they can keep these forever

Reply:
Well, I agree with all of this, but 

1) whatever `localhost:8234` returns is good, since we don't want any cache in the local dev server
2) whatever `localhost:8000` returns might be ignored anyway. Also, is this django after a full build and `collectstatic`, running [in production mode](https://github.com/PostHog/posthog/blob/9a4062e289da1fe5b770e76e968a4183424e6d25/posthog/settings.py#L264)? 

On app.posthog.com, a single reload definitely does not make network requests to fetch these static assets:

![image](https://user-images.githubusercontent.com/53387/140907540-a1bf2127-71c9-4f65-902b-173b7dfc1486.png)

There's also another issue here. Django's `collectstatic` re-hashes the files, and actually publishes them twice. Once as they are, and once with another `-someotherhash.js` appended to the end... 🤔

Reply:
![the-simpsons-homer-simpson](https://user-images.githubusercontent.com/984817/140910082-0361b708-67e4-43ba-ac04-dcf64819f0ad.gif)